syntax				= "proto3";
option optimize_for = CODE_SIZE;

message Cfg_DataCountLimit
{

		uint32 type		  = 1;
		uint32 keyidx	  = 2;
		uint32 max_count  = 3;
		uint32 reset_type = 4;
		uint32 reset_time = 5;

}